2019/serial/D/Makefile

```diff
@@ -0,0 +1,15 @@
+# Makefile for simulation
+
+LDFLAGS=-lm
+CFLAGS = -g -Wall -Wno-deprecated -O3 -march=native -std=c11
```

Comment:
Make sure that no serial problem is being built with `-g` or `-O0` flags. Replace them with `-O2` flags